pkg/controller/plan/kubevirt.go

```diff
@@ -1794,7 +1795,12 @@ func (r *KubeVirt) createOpenStackVolumes(vm ref.Ref) (pvcNames []string, err er
 				return
 			}
 
-			populatorCr := openstackutil.OpenstackVolumePopulator(image, sourceUrl, r.Plan.Spec.TargetNamespace, secret.Name, r.Migration.Name)
+			if image.Status != "active" {
+				r.Log.Info("Image is not active yet", "image", image.Name)
+				continue
+			}
```

Comment:
I think this is not part of the PR that was merged to the main branch, do we want this here?